LayoutTests/fast/repaint/list-marker-expected.txt

```diff
@@ -0,0 +1,48 @@
+layer at (0,0) size 800x600
```

Comment:
I don't think this is needed since you have a ref test setup here.

Reply:
oh, my bad.

Reply:
but this is a new file, I am confused.

Reply:
ok, makes sense. if all platforms match, you can remove the platform specific tree dumps.